app/_gateway_entities/vault.md

```diff
@@ -0,0 +1,146 @@
+        Those secrets would be unrecoverable.
+  - q: How should I manage my Vault configuration with decK?
+    a: |
+        To keep your environment secure and avoid taking down your proxies by accident, make sure to:
+        * Manage Vaults with distributed configuration via tags.
+        * Use a separate [RBAC role, user, and token](/gateway/entities/rbac/)
+        to manage Vaults. Don't use a generic admin user.
+        * Set up a separate CI pipeline for Vaults.
+
+tools:
+    - admin-api
+    - konnect-api
+    - kic
+    - deck
+    - terraform
+
+api_specs:
+    - gateway/admin-oss
+    - gateway/admin-ee
+    - konnect/control-planes-config
+
+schema:
+    api: gateway/admin-ee
+    path: /schemas/Vault
+---
+
+## What is a Vault?
+In {{site.base_gateway}}, the Vault object is used to store secrets. A secret is any sensitive piece of information required for API gateway
+operations. Secrets can be used as part of the core {{site.base_gateway}} configuration, in plugins, or in configuration associated
+with APIs serviced by the gateway.
+
+Some of the most common types of secrets used by {{site.base_gateway}} include:
+
+* Data store usernames and passwords, used with PostgreSQL and Redis
+* Private X.509 certificates
+* API keys
+* Sensitive plugin configuration fields, generally used for authentication, hashing, signing, or encryption
+
+## Vault use cases
+
+The main use case for Vaults is that they allow you to securely store and then reference those secrets from other entities. This ensures that secrets aren't visible in plaintext throughout the platform, in places such as `kong.conf`,
+in declarative configuration files, logs, or in the UI.
+
+For example, you could store a certificate and a key in a Vault, then reference them from a [Certificate entity](/gateway/entities/certificate/). This way, the certificate and key are not stored in the entity directly and are more secure.
+
+## How do I add secrets to a Vault?
+
+Vaults store secrets, they don't create secrets. You must add secrets to the Vault in one of the following ways: 
+* Environment variables
+* {{site.konnect_short_name}} Config Store
+* Supported third-party backend vault
+
+The following table goes into detail about the different options you have:
+
+| Backend                                                   | {{site.base_gateway}} OSS | {{site.base_gateway}} Enterprise | Uses Vault entity              | {{site.konnect_short_name}} supported |
+| --------------------------------------------------------- | ------------------------ | -------------------------------- | ------------------------------ | ------------------------------------- |
+| [Environment variable](/how-to/store-secrets-as-env-variables) | <img src="/app/assets/icons/check.svg" alt="Check icon">   | <img src="/app/assets/icons/check.svg" alt="Check icon">   | <img src="/app/assets/icons/close.svg" alt="Cross icon">   | <img src="/app/assets/icons/check.svg" alt="Check icon">        |
```

Comment:
The check marks aren't rendering, does anyone know what I did wrong?

Reply:
I just pushed a commit to your branch, please double-check if the data is correct.
I added `{% feature_table %}` block (it works in the same way as the component we use on landing pages), it takes a yaml config and renders the table